### ISSUE TITLE
Ticket not print the TOTAL

### ISSUE THREAD

**Issue by serdirep** [star-micronics/star-cloudprnt-for-woocommerce#31]
I have the printer C3-printer in woocommerce but not print the total in the ticket, display 0.00€ . It also does not print the customer's data or the payment method. I attach an example ticket and the php file where I have only modified the name of the tags but not the variables. Let's see if you can give me a hand please. Thank you so much.
![lng1v136xcjrhoxnqthtqhngzsry0wf2s_q2aggoqxo=](https://github.com/star-micronics/star-cloudprnt-for-woocommerce/assets/161164915/1255093b-cae2-4182-b249-b2154e985635)
[order-handler (1) modificado.zip](https://github.com/star-micronics/star-cloudprnt-for-woocommerce/files/14394401/order-handler.1.modificado.zip)
![lng1v136xcjrhoxnqthtqrwgjlipyhxpvzybmstk7xw=](https://github.com/star-micronics/star-cloudprnt-for-woocommerce/assets/161164915/d51aef13-cf0c-42ae-82bd-3429cf938331)

**Comment by Chlitzxer** [comment]
@serdirep were you able to fix this issue? I am looking for this plugin funcional and I haven't found anything yet :(

**Comment by ti11x** [comment]
@Chlitzxer @serdirep @Has anyone been able to fix this issue?

**Comment by Chlitzxer** [comment]
> [@Chlitzxer](https://github.com/Chlitzxer) [@serdirep](https://github.com/serdirep) [@Has](https://github.com/Has) anyone been able to fix this issue?

No, I gave up on this and not using this printers anymore. There is a guy here charging for a plugin that cost more than a used printer lol

**Comment by ti11x** [comment]
@Chlitzxer I finally fixed it using AI :-)

**Comment by Chlitzxer** [comment]
> [@Chlitzxer](https://github.com/Chlitzxer) I finally fixed it using AI :-)

That is great you should publish your version as it may help many people that need this ;)

**Comment by ti11x** [comment]
My order_handler.php layout is customized to my own needs, not sure it would help others that much.
